kernel/enter.c

```diff
@@ -446,15 +470,26 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     }
     else
     {
-#ifdef MYST_ENABLE_EXT2FS
```

Comment:
Can we move the file system mounting operations to a separate function?

Reply:
Good idea. I moved it to **_mount_rootfs()**.